### DISCUSSION TITLE
Keyball44でトラックボールが反応しなくなりました

### DISCUSSION THREAD

**Discussion by izumeee** [Yowkees/keyball#406] [Q&A]
10ヶ月程問題なく使えていたのですが、1ヶ月前くらいからトラックボールが全く反応しなくなるり、トラックボール読取り基板を外して付け直してみたり、関係ありそうな半田を足してみたりしているうちに正常に動作するようになる、といったことが何度かあったのですが、ついに何をしても動かない状態になりました。
他のディスカッションを参考に、メイン基板の通電確認は行って、色がついている箇所の通電ができていることは確認済みです。
そのためトラックボール読取り基板側の問題ではないかと考えています。
半田を何度もやり直してみたので悪化させてしまった気もしますが、今の状態の写真を載せておきます。
また、他の方の投稿を見て気になったのですが、トラックボールの読み取り部分(?)が溶けているように見えます。
以前からこの状態だった気がするのですが、問題ありますでしょうか。

![275303563-e7524b66-1f61-4348-83d9-5efd48a7ab5d](https://github.com/Yowkees/keyball/assets/108213174/9bd972ce-90ad-4258-bc9c-42c68e275182)

![2023-11-02 01 55 26](https://github.com/Yowkees/keyball/assets/108213174/6f2ab4fd-0d80-4ffb-a90c-6eb5544119b3)

![2023-11-02 01 55 55](https://github.com/Yowkees/keyball/assets/108213174/abb5df4c-1620-4812-b04b-e0cd18f5e0a6)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

溶けてしまっている個所は、本来はピンホールになっておりますので穴が大きくなってしまっています。半田付け作業でコテが触ってしまったでしょうか。動いていた頃もセンサー本来の精度は出ていなかったかも知れません。

白銀ラボウェブショップにてKeyballシリーズ購入者様限定で、半田付けして動作確認済のトラックボール基板を販売しておりますので、ご希望でしたら下記白銀ラボのお問合せよりご連絡いただければ幸いです。
https://shirogane-lab.net/inquiry

以上、ご検討よろしくお願いいたします。

**Reply by izumeee** [reply]
こちら購入した基盤に交換したところ正常に動作するようになりました。
ありがとうございました。

**Comment by Yowkees** [comment]
無事に直ったようで良かったです！安心しました。
ぜひ末永くご愛用いただければ幸いです。
